opencood/hypes_yaml/point_pillar_where2comm.yaml

```diff
@@ -1,73 +1,72 @@
 name: point_pillar_where2comm
 root_dir: '/home/cav/data/v2xset/train'
-validate_dir: '/home/cav/data/v2xset//test'
+validate_dir: '/home/cav/data/v2xset/test'
 
 
 wild_setting:
-  async: false
+  seed: 42
```

Comment:
I used seed 20 in V2X-ViT paper, so let's keep the same to be consistent with the paper for reproducible results

Reply:
You are right. Keep 20 for fair comparison.